backend/src/main/resources/graphql/main.graphqls

```diff
@@ -383,6 +383,8 @@ type Query {
     pageSize: Int = 5000
     includeArchived: Boolean = false
```

Comment:
⚠️ for backward compatibility -- will deprecate in separate PR once this one is deployed